.storybook/recipes/CalendarCard/CalendarCard.tsx

```diff
@@ -0,0 +1,54 @@
+import clsx from 'clsx';
+import React, { ReactNode } from 'react';
+import styles from './CalendarCard.module.css';
+
+import { Card } from '../../../src';
+
+export const VARIANTS = ['brand', 'revise', 'success'] as const;
+
+export type Variant = typeof VARIANTS[number];
+
+export interface Props {
+  /**
+   * CSS class names that can be appended to the component.
+   */
+  className?: string;
+  /**
+   * Card sub components that are nested within the Card wrapper.
+   */
+  children: ReactNode;
+  /**
+   * Notification variant for the card.
+   */
+  variant?: Variant;
+}
+
+/**
+ * Recipe for a Card component to represent an event within calendar UI
+ */
+export const CalendarCard = ({
+  children,
+  className,
+  variant = 'brand',
+  ...other
+}: Props) => {
+  const componentClassName = clsx(
+    styles['calendar-card'],
+    className,
+    styles[`calendar-card--${variant}`],
+  );
+  return (
+    <div className={componentClassName}>
+      <Card className={styles['calendar-card__card']} {...other}>
```

Comment:
I don't see `calendar-card__card` being defined anywhere. Could we put `componentClassName` here and drop the wrapping `div`? The border looks a little detached right now because it's on the wrapper `div` around the card. And then you could drop the `border-radius` on this component so we don't have to worry about this getting out of sync with `Card`.

<img width="137" alt="calendar card in storybook, zoomed in and cropped to only show the left side" src="https://user-images.githubusercontent.com/7761701/173161533-81a76e60-a6ca-4018-b736-29371a2e55ec.png">